bin/runmigration.js

```diff
@@ -1,88 +1,115 @@
+  },
+  {
+    name: 'list', alias: 'l', type: Boolean, description: 'Show migration file list (without execution)', defaultValue: false,
+  },
+  { name: 'help', type: Boolean, description: 'Show this message' },
 ];
 
 const options = commandLineArgs(optionDefinitions);
 
-let migrationsDir = path.join(process.env.PWD, options['migrations-path'] || 'migrations'),
-    modelsDir     = path.join(process.env.PWD, options['models-path'] || 'models');
 
-if (options.help)
-{
-    console.log("Simple sequelize migration execution tool\n\nUsage:");
-    optionDefinitions.forEach((option) => {
-        let alias = (option.alias) ? ` (-${option.alias})` : '\t';
-        console.log(`\t --${option.name}${alias} \t${option.description}`);
-    });
-    process.exit(0);
+if (options.help) {
+  console.log('Simple sequelize migration execution tool\n\nUsage:');
+  optionDefinitions.forEach((option) => {
+    const alias = (option.alias) ? ` (-${option.alias})` : '\t';
+    console.log(`\t --${option.name}${alias} \t${option.description}`);
+  });
+  process.exit(0);
 }
 
-const Sequelize = require("sequelize");
-const sequelize = require(modelsDir).sequelize;
+const { sequelize } = require(modelsDir);
 const queryInterface = sequelize.getQueryInterface();
 
 // execute all migration from
-let fromRevision = options.rev;
-let fromPos = parseInt(options.pos);
-let stop = options.one;
-
-let migrationFiles = fs.readdirSync(migrationsDir)
-// filter JS files
-  .filter((file) => {
-    return (file.indexOf('.') !== 0) && (file.slice(-3) === '.js');
-  })
-// sort by revision
-  .sort( (a, b) => {
-      let revA = parseInt( path.basename(a).split('-',2)[0]),
-          revB = parseInt( path.basename(b).split('-',2)[0]);
-      if (revA < revB) return -1;
-      if (revA > revB) return 1;
-      return 0;
-  })
-// remove all migrations before fromRevision
-  .filter((file) => {
-      let rev = parseInt( path.basename(file).split('-',2)[0]);
-      return (rev >= fromRevision);
-  });
-  
-console.log("Migrations to execute:");  
-migrationFiles.forEach((file) => {
-    console.log("\t"+file);
-});
-
-if (options.list)
-    process.exit(0);
+const fromRevision = options.rev;
+let fromPos = parseInt(options.pos, 10);
+const stop = options.one;
 
-
-Async.eachSeries(migrationFiles, 
-    function (file, cb) {
-        console.log("Execute migration from file: "+file);
-        migrate.executeMigration(queryInterface, path.join(migrationsDir, file), fromPos, (err) => {
-            if (stop)
-                return cb("Stopped");
-                
-            cb(err);
+queryInterface.createTable('SequelizeMeta', {
+  name: {
+    type: Sequelize.STRING,
+    allowNull: false,
+    unique: true,
+    primaryKey: true,
+  },
+}).then(() => {
+  const alreadyExecuted = [];
+  try {
+    sequelize.query('SELECT * FROM "SequelizeMeta"', { type: sequelize.QueryTypes.SELECT })
```

Comment:
is `SELECT * FROM "SequelizeMeta"` correctly?

i think it should be ``SELECT * FROM `SequelizeMeta` ``